src/utils/alt_bn128/bigint.cairo

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
```

Comment:
is this copied from elsewhere?

Reply:
It's the [alt_bn128](https://github.com/tekkac/cairo-alt_bn128) lib from tekkac, I had the reference in the main file but just added to all files